README.md

```diff
@@ -50,14 +50,14 @@ Feel free to re-use code in DSharpPlus.
 ## NuGet
 Package|NuGet
 |--|--|
-DSharpPlusNextGen|[![NuGet](https://img.shields.io/nuget/vpre/DSharpPlusNextGen.svg?label=)](https://nuget.dspng.aitsys.dev/DSharpPlusNextGen)
+DisCatSharp|[![NuGet](https://img.shields.io/nuget/vpre/DSharpPlusNextGen.svg?label=)](https://nuget.dspng.aitsys.dev/DSharpPlusNextGen)
 DSharpPlusNextGen.CommandsNext|[![NuGet](https://img.shields.io/nuget/vpre/DSharpPlusNextGen.CommandsNext.svg?label=)](https://nuget.dspng.aitsys.dev/DSharpPlusNextGen.CommandsNext)
```

Comment:
CommandsNext wasn't renamed? Is this because it's going to be chopped or was it missed on accident?

Reply:
2nd.

Reply:
fixed